backend/backend.go

```diff
@@ -49,7 +49,8 @@ type Backend interface {
 	DeleteTaskHub(context.Context) error
 
 	// Start starts any background processing done by this backend.
-	Start(context.Context) error
+	// Backends supporting this feature can filter orchestrator and activity types server-side.
+	Start(ctx context.Context, orchestrators *[]string, activities *[]string) error
```

Comment:
Why a pointer to a slice? This is unusual. Do you have to modify the slice?

Reply:
I keep forgetting nil is a value I can use for slices :) No reason other than the ability to check for nil / use nil.